configs/eln_extras_rhel_packager.yaml

```diff
@@ -3,7 +3,7 @@ version: 1
 data:
   name: RHEL packager
   description: ELN Extras packages for RHEL maintainers
-  maintainer: carlwgeorge
+  maintainer: rhel-sst-cpe
```

Comment:
After the QE merger, the CPE team is now the CLE team.

https://docs.fedoraproject.org/en-US/cle/

Regardless of that, I'm not aware that the CPE team was ever considered a RHEL SST, and the same goes for CLE.  I'm happy to hand off ownership of this to whoever makes the most sense, but it should probably be to a real SST.